src/components/TouchCapture/src/TouchCapture.vue

```diff
@@ -107,6 +111,9 @@ export default {
 
 	methods: {
 		handleTouchEvent(event) {
+			if (this.preventDefault) {
+				event.preventDefault();
```

Comment:
Based on the description in the ticket, it actually sounds like you want `event.stopPropagation()` instead of `preventDefault`.

Reply:
I tested this but it doesn't look like `event.stopPropagation` does the trick here because scrolling is a default behavior which stopPropagation doesn't prevent.
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation

Reply:
TIL!